mobile/src/components/common/input/input.tsx

```diff
@@ -29,22 +29,39 @@ const Input = <T extends FormControlValues>({
   placeholder,
 }: Props<T>): ReactElement => {
   const { field } = useFormControl({ name, control });
+  const [isSecure, setisSecure] = useState(name === 'password');
 
   const { value, onChange, onBlur } = field;
   const error = errors[name]?.message as string;
 
+  const handleOnPress = (): void => {
+    setisSecure(!isSecure);
+  };
+
   return (
     <View>
       <Text style={styles.label}>{label}</Text>
-      <TextInput
-        value={value}
-        placeholder={placeholder}
-        placeholderTextColor={AppColor.TEXT.GRAY_200}
-        onChangeText={onChange}
-        onBlur={onBlur}
-        style={styles.input}
-        secureTextEntry={name === 'password'}
-      />
+      <View>
+        <TextInput
+          value={value}
+          placeholder={placeholder}
+          placeholderTextColor={AppColor.TEXT.GRAY_200}
+          onChangeText={onChange}
+          onBlur={onBlur}
+          style={styles.input}
+          secureTextEntry={isSecure}
+        />
+        {name === 'password' && (
```

Comment:
I think it`s not a good idea to depend on name, bcos for example input name has a name - confirmPassword. So please app a type prop: type: 'text' | 'password'